detectable/src/main/java/com/synopsys/integration/detectable/detectables/conan/cli/ConanCliExtractor.java

```diff
@@ -78,19 +78,4 @@ private boolean producedOutput(ExecutableOutput conanInfoOutput) {
         return true;
     }
 
-    @NotNull
-    private List<String> generateConanInfoCmdArgs(File projectDir, ConanCliExtractorOptions conanCliExtractorOptions) {
```

Comment:
I moved this to it's own class that only take in the config needed. `ConanArgumentGenerator`